pkg/dss/server.go

```diff
@@ -298,6 +298,14 @@ func (s *Server) createOrUpdateSubscription(
 	if extents == nil {
 		return nil, dsserr.BadRequest("missing required extents")
 	}
+	var (
+		endTime   = time.Unix(extents.TimeEnd.GetSeconds(), 0)
+		startTime = time.Unix(extents.TimeStart.GetSeconds(), 0)
+		diff      = endTime.Sub(startTime)
+	)
+	if diff.Hours() > 24 {
```

Comment:
Rejecting Subscription requests longer than 24 hours is fine, as is truncating them to 24 hours.  To use the rejection approach, let's remove [the now-redundant truncation code](https://github.com/interuss/dss/blob/976a7f16a9006c6e15b17cf21cfe37267f491bbb/pkg/dss/models/subscriptions.go#L137) and let's declare the "24" magic number as a constant

Reply:
I've exported models.MaxSubscriptionDuration and removed the truncation clause of the check, but kept the rest so that it could default to 1Day when end time is not supplied